### PR TITLE
Refactor transaction API to support transactions query and  save sequencer last_order to store

### PR DESCRIPTION
**Pull request by baichuan3**
resolve https://github.com/rooch-network/rooch/issues/754 and https://github.com/rooch-network/rooch/issues/763

Summary
1, Aggregate and simplify transaction RPC,  provide both `getTransactionsByOrder` for scanning the tx list and `getTransactionsByHash` for multi-get by hash.
 - Define `TransactionReturn`, including transaction, sequence_info and execution_info, aggregate returns in `getTransactionsByOrder` API
 ```
pub struct TransactionReturn {
    pub transaction: TypedTransaction,
    pub sequence_info: TransactionSequenceInfo,
    pub execution_info: TransactionExecutionInfo,
}
```
- Unify all transaction API name
- Remove `getTransactionInfosByTxHash` API
-  For SDK compatibility and other scenarios, the interface `getTransactionByHash` is still retained. If the API is used infrequently, it may be considered to be removed.

2, Save sequencer last_order to rooch store, meanwhile check `last_order` when server restart.

3, The flow of obtaining transaction list through Explorer or Dashboard:
Call the `rooch_getTransactionsByOrder` API to obtain the transaction list (transaction hash) according to sequence order.

```
curl --location 'http://localhost:50051' \
--header 'Content-Type: application/json' \
--data '{
 "id":101, 
 "jsonrpc":"2.0", 
 "method":"rooch_getTransactionsByOrder", 
 "params":[null, null]
}'
```

```
{
    "jsonrpc": "2.0",
    "result": {
        "data": [
            {
                "transaction": {
                    "transaction_type": "rooch",
                    "sequence_number": 0,
                    "sender": "0x583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e",
                    "action_type": "functioncall",
                    "action": {
                        "function_call": {
                            "function_id": "0000000000000000000000000000000000000000000000000000000000000002::account_storage::publish_modules_entry",
                            "ty_args": [],
                            "args": [
                                "0x02ee05a11ceb0b060000000a01000c020c12031e36045408055c4f07ab01d20108fd02800106fd03210a9e04050ca3049b010000010101020203020403050006080004070000050c040108010210070000080001000009020100000a030100030d050600050e08090108050f0a01010802110b0c0005120d01010801130e0e0005140f010108040705070707090702070801060c000107080104070801060c050f02050b0201080001060c0105010800020708010f010b0201090003070801050b02010900010a0201080304070801080308030201030407080105050f0c707269766174655f636f696e056572726f7206737472696e67067369676e65720f73746f726167655f636f6e7465787404636f696e035052430e53746f72616765436f6e746578740666617563657404696e6974087472616e736665720b64756d6d795f6669656c6404436f696e0a616464726573735f6f660b6d696e745f657874656e640e6465706f7369745f657874656e6406537472696e6704757466380f72656769737465725f657874656e6410696e76616c69645f617267756d656e740f7472616e736665725f657874656e64583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000003030801000000000000000a020d0c5072697661746520436f696e0a0204035052430002010b0100010400040c0b0111030c020a004a102700000000000000000000000000000000000000000000000000000000000038000c030b000b020b033801020100000001080b00070111060702110631013802020201040006150a034a6400000000000000000000000000000000000000000000000000000000000000250405050c0b01010b000107001108270b0111030c040b000b040b020b0338030200d106a11ceb0b060000000b01000c020c1603223e04600e056e6307d101f00108c103800106c104600aa1050e0caf056d0d9c060200000101020202030204030500060c000007080004080000050c04010801011107000009000100000a020100030d040500020e07080108050f0a0b010805100c01010c01120e0f000513100101080314011101000515120b01080216130101080306040905090709080909090a0602070802060c000107080202050b0301080001060c010501080102070802050107090001080002070b030109000f010b0301090003070802050b03010900020b030108000c010a02010804040708020804080402010c020708020f03070802060c09001166697865645f737570706c795f636f696e06737472696e670f6163636f756e745f73746f72616765067369676e65720f73746f726167655f636f6e7465787404636f696e034653430854726561737572790e53746f72616765436f6e746578740666617563657404696e69740b64756d6d795f6669656c6404436f696e0a616464726573735f6f6611676c6f62616c5f626f72726f775f6d75740765787472616374076465706f73697406537472696e6704757466380f72656769737465725f657874656e640d6d6f64756c655f7369676e65720b6d696e745f657874656e640e676c6f62616c5f6d6f76655f746f583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000030f2000b4f9e4300000000000000000000000000000000000000000000000000000000520583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e0a021211466978656420537570706c7920436f696e0a0204034653430002010b01010201050b0301080000010400030f0b0111020c020a00070138000f004a102700000000000000000000000000000000000000000000000000000000000038010c030b000b020b03380202010000000d130a0007021106070311063101380338040c020a00070038050c010b000e020b011201380602010000"
                            ]
                        }
                    },
                    "raw": "0x583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e0000000000000000d7af34010000000000e1f505000000000100000000000000000000000000000000000000000000000000000000000000020f6163636f756e745f73746f72616765157075626c6973685f6d6f64756c65735f656e7472790001c40c02ee05a11ceb0b060000000a01000c020c12031e36045408055c4f07ab01d20108fd02800106fd03210a9e04050ca3049b010000010101020203020403050006080004070000050c040108010210070000080001000009020100000a030100030d050600050e08090108050f0a01010802110b0c0005120d01010801130e0e0005140f010108040705070707090702070801060c000107080104070801060c050f02050b0201080001060c0105010800020708010f010b0201090003070801050b02010900010a0201080304070801080308030201030407080105050f0c707269766174655f636f696e056572726f7206737472696e67067369676e65720f73746f726167655f636f6e7465787404636f696e035052430e53746f72616765436f6e746578740666617563657404696e6974087472616e736665720b64756d6d795f6669656c6404436f696e0a616464726573735f6f660b6d696e745f657874656e640e6465706f7369745f657874656e6406537472696e6704757466380f72656769737465725f657874656e6410696e76616c69645f617267756d656e740f7472616e736665725f657874656e64583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000003030801000000000000000a020d0c5072697661746520436f696e0a0204035052430002010b0100010400040c0b0111030c020a004a102700000000000000000000000000000000000000000000000000000000000038000c030b000b020b033801020100000001080b00070111060702110631013802020201040006150a034a6400000000000000000000000000000000000000000000000000000000000000250405050c0b01010b000107001108270b0111030c040b000b040b020b0338030200d106a11ceb0b060000000b01000c020c1603223e04600e056e6307d101f00108c103800106c104600aa1050e0caf056d0d9c060200000101020202030204030500060c000007080004080000050c04010801011107000009000100000a020100030d040500020e07080108050f0a0b010805100c01010c01120e0f000513100101080314011101000515120b01080216130101080306040905090709080909090a0602070802060c000107080202050b0301080001060c010501080102070802050107090001080002070b030109000f010b0301090003070802050b03010900020b030108000c010a02010804040708020804080402010c020708020f03070802060c09001166697865645f737570706c795f636f696e06737472696e670f6163636f756e745f73746f72616765067369676e65720f73746f726167655f636f6e7465787404636f696e034653430854726561737572790e53746f72616765436f6e746578740666617563657404696e69740b64756d6d795f6669656c6404436f696e0a616464726573735f6f6611676c6f62616c5f626f72726f775f6d75740765787472616374076465706f73697406537472696e6704757466380f72656769737465725f657874656e640d6d6f64756c655f7369676e65720b6d696e745f657874656e640e676c6f62616c5f6d6f76655f746f583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000030f2000b4f9e4300000000000000000000000000000000000000000000000000000000520583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e0a021211466978656420537570706c7920436f696e0a0204034653430002010b01010201050b0301080000010400030f0b0111020c020a00070138000f004a102700000000000000000000000000000000000000000000000000000000000038010c030b000b020b03380202010000000d130a0007021106070311063101380338040c020a00070038050c010b000e020b011201380602010000000000000000000061001a0fd6644299e190d86bf7dc9d85b40abc97a2489772a89a8b514c4c1a6838bb62984a090aebf193c14770981c633dfc5eb91edf2650478ebe86f35c1b41b60c07b50f0bebaee09ff0edd1d41b9a03eb80ac91e366dd86abf65072d10f86bd7f"
                },
                "sequence_info": {
                    "tx_order": "1",
                    "tx_order_signature": {
                        "coin_id": "0",
                        "payload": "0x00bc593e768dfa225dbba7c13e8e1793defc0fd6857ab7be9457324ee76c91a3edd7708296064e1af22d1164131827855698269ffff5e9dcfba080f7727638d30899da677a6d98384c838deb162332669db9af8d02770cab306952695e29cadddc"
                    },
                    "tx_accumulator_root": "0xc862bc8ae2d0b8091148445c8075e63df0a7496f4c1f7eccd6b7fa1da68f25d8"
                },
                "execution_info": {
                    "tx_hash": "0x40ec712fe17fd1a4bbcfa511319774518633f4b0eeae4e8d10995ab023e44c6f",
                    "state_root": "0x90d92109d242c3db1548c171ffc1218cd764dfdd4d502ce5e9c4156a626205e5",
                    "event_root": "0x7e8b160d16e2f6efcec8b08630da2aa5ffca89417db98f317e04f78099e769a6",
                    "gas_used": 0,
                    "status": {
                        "type": "executed"
                    }
                }
            },
            {
                "transaction": {
                    "transaction_type": "rooch",
                    "sequence_number": 1,
                    "sender": "0x583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e",
                    "action_type": "functioncall",
                    "action": {
                        "function_call": {
                            "function_id": "583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e::fixed_supply_coin::faucet",
                            "ty_args": [],
                            "args": []
                        }
                    },
                    "raw": "0x583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e0100000000000000d7af34010000000000e1f5050000000001583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e1166697865645f737570706c795f636f696e06666175636574000000000000000000006100a25a8b368234cdd8ed03ba6c595c067ab65812f3b2dc38741d2afb8ebb1c53167126977fdab5fb19ea5eb2314cbca20d75ea13be513cb54419e7f3811ece400407b50f0bebaee09ff0edd1d41b9a03eb80ac91e366dd86abf65072d10f86bd7f"
                },
                "sequence_info": {
                    "tx_order": "2",
                    "tx_order_signature": {
                        "coin_id": "0",
                        "payload": "0x00829e17b84b707b4ef298a42824ed8d6bfb85ed1bbe02cc9d23af769729e2acdd4f6d60cc9ebf98a5bc62cda9d2272df307b795e2c4c952f19def83df459d670499da677a6d98384c838deb162332669db9af8d02770cab306952695e29cadddc"
                    },
                    "tx_accumulator_root": "0x330b12c19c9bc6d02d5b9b1bc68a6844060a1cb63a21f9c8a4716c4af5f7c996"
                },
                "execution_info": {
                    "tx_hash": "0x60185a08d4b77296a4103707676f5d8ccf5584f524cbdd6aa61bcfba5a7385ec",
                    "state_root": "0x3601ac6287f3d3c94abe19cd9da89e80dc40be83add098d52b6346c57bce50df",
                    "event_root": "0x02513da8851b4e4bb86f5e5782983413bfdc08c18bc2b341f37bf92656a23a8f",
                    "gas_used": 0,
                    "status": {
                        "type": "executed"
                    }
                }
            }
        ],
        "next_cursor": 2,
        "has_next_page": false
    },
    "id": 101
}
```

or Call the `rooch_getTransactionsByHash` API to obtain the Transaction list in batches.

```
curl --location 'http://localhost:50051' \
--header 'Content-Type: application/json' \
--data '{
 "id":101, 
 "jsonrpc":"2.0", 
 "method":"rooch_getTransactionsByHash", 
 "params":[["0x40ec712fe17fd1a4bbcfa511319774518633f4b0eeae4e8d10995ab023e44c6f", "0x60185a08d4b77296a4103707676f5d8ccf5584f524cbdd6aa61bcfba5a7385ec"]]
}'
```

```
{
    "jsonrpc": "2.0",
    "result": [
        {
            "transaction_type": "rooch",
            "sequence_number": 0,
            "sender": "0x583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e",
            "action_type": "functioncall",
            "action": {
                "function_call": {
                    "function_id": "0000000000000000000000000000000000000000000000000000000000000002::account_storage::publish_modules_entry",
                    "ty_args": [],
                    "args": [
                        "0x02ee05a11ceb0b060000000a01000c020c12031e36045408055c4f07ab01d20108fd02800106fd03210a9e04050ca3049b010000010101020203020403050006080004070000050c040108010210070000080001000009020100000a030100030d050600050e08090108050f0a01010802110b0c0005120d01010801130e0e0005140f010108040705070707090702070801060c000107080104070801060c050f02050b0201080001060c0105010800020708010f010b0201090003070801050b02010900010a0201080304070801080308030201030407080105050f0c707269766174655f636f696e056572726f7206737472696e67067369676e65720f73746f726167655f636f6e7465787404636f696e035052430e53746f72616765436f6e746578740666617563657404696e6974087472616e736665720b64756d6d795f6669656c6404436f696e0a616464726573735f6f660b6d696e745f657874656e640e6465706f7369745f657874656e6406537472696e6704757466380f72656769737465725f657874656e6410696e76616c69645f617267756d656e740f7472616e736665725f657874656e64583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000003030801000000000000000a020d0c5072697661746520436f696e0a0204035052430002010b0100010400040c0b0111030c020a004a102700000000000000000000000000000000000000000000000000000000000038000c030b000b020b033801020100000001080b00070111060702110631013802020201040006150a034a6400000000000000000000000000000000000000000000000000000000000000250405050c0b01010b000107001108270b0111030c040b000b040b020b0338030200d106a11ceb0b060000000b01000c020c1603223e04600e056e6307d101f00108c103800106c104600aa1050e0caf056d0d9c060200000101020202030204030500060c000007080004080000050c04010801011107000009000100000a020100030d040500020e07080108050f0a0b010805100c01010c01120e0f000513100101080314011101000515120b01080216130101080306040905090709080909090a0602070802060c000107080202050b0301080001060c010501080102070802050107090001080002070b030109000f010b0301090003070802050b03010900020b030108000c010a02010804040708020804080402010c020708020f03070802060c09001166697865645f737570706c795f636f696e06737472696e670f6163636f756e745f73746f72616765067369676e65720f73746f726167655f636f6e7465787404636f696e034653430854726561737572790e53746f72616765436f6e746578740666617563657404696e69740b64756d6d795f6669656c6404436f696e0a616464726573735f6f6611676c6f62616c5f626f72726f775f6d75740765787472616374076465706f73697406537472696e6704757466380f72656769737465725f657874656e640d6d6f64756c655f7369676e65720b6d696e745f657874656e640e676c6f62616c5f6d6f76655f746f583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000030f2000b4f9e4300000000000000000000000000000000000000000000000000000000520583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e0a021211466978656420537570706c7920436f696e0a0204034653430002010b01010201050b0301080000010400030f0b0111020c020a00070138000f004a102700000000000000000000000000000000000000000000000000000000000038010c030b000b020b03380202010000000d130a0007021106070311063101380338040c020a00070038050c010b000e020b011201380602010000"
                    ]
                }
            },
            "raw": "0x583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e0000000000000000d7af34010000000000e1f505000000000100000000000000000000000000000000000000000000000000000000000000020f6163636f756e745f73746f72616765157075626c6973685f6d6f64756c65735f656e7472790001c40c02ee05a11ceb0b060000000a01000c020c12031e36045408055c4f07ab01d20108fd02800106fd03210a9e04050ca3049b010000010101020203020403050006080004070000050c040108010210070000080001000009020100000a030100030d050600050e08090108050f0a01010802110b0c0005120d01010801130e0e0005140f010108040705070707090702070801060c000107080104070801060c050f02050b0201080001060c0105010800020708010f010b0201090003070801050b02010900010a0201080304070801080308030201030407080105050f0c707269766174655f636f696e056572726f7206737472696e67067369676e65720f73746f726167655f636f6e7465787404636f696e035052430e53746f72616765436f6e746578740666617563657404696e6974087472616e736665720b64756d6d795f6669656c6404436f696e0a616464726573735f6f660b6d696e745f657874656e640e6465706f7369745f657874656e6406537472696e6704757466380f72656769737465725f657874656e6410696e76616c69645f617267756d656e740f7472616e736665725f657874656e64583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000003030801000000000000000a020d0c5072697661746520436f696e0a0204035052430002010b0100010400040c0b0111030c020a004a102700000000000000000000000000000000000000000000000000000000000038000c030b000b020b033801020100000001080b00070111060702110631013802020201040006150a034a6400000000000000000000000000000000000000000000000000000000000000250405050c0b01010b000107001108270b0111030c040b000b040b020b0338030200d106a11ceb0b060000000b01000c020c1603223e04600e056e6307d101f00108c103800106c104600aa1050e0caf056d0d9c060200000101020202030204030500060c000007080004080000050c04010801011107000009000100000a020100030d040500020e07080108050f0a0b010805100c01010c01120e0f000513100101080314011101000515120b01080216130101080306040905090709080909090a0602070802060c000107080202050b0301080001060c010501080102070802050107090001080002070b030109000f010b0301090003070802050b03010900020b030108000c010a02010804040708020804080402010c020708020f03070802060c09001166697865645f737570706c795f636f696e06737472696e670f6163636f756e745f73746f72616765067369676e65720f73746f726167655f636f6e7465787404636f696e034653430854726561737572790e53746f72616765436f6e746578740666617563657404696e69740b64756d6d795f6669656c6404436f696e0a616464726573735f6f6611676c6f62616c5f626f72726f775f6d75740765787472616374076465706f73697406537472696e6704757466380f72656769737465725f657874656e640d6d6f64756c655f7369676e65720b6d696e745f657874656e640e676c6f62616c5f6d6f76655f746f583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000030f2000b4f9e4300000000000000000000000000000000000000000000000000000000520583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e0a021211466978656420537570706c7920436f696e0a0204034653430002010b01010201050b0301080000010400030f0b0111020c020a00070138000f004a102700000000000000000000000000000000000000000000000000000000000038010c030b000b020b03380202010000000d130a0007021106070311063101380338040c020a00070038050c010b000e020b011201380602010000000000000000000061001a0fd6644299e190d86bf7dc9d85b40abc97a2489772a89a8b514c4c1a6838bb62984a090aebf193c14770981c633dfc5eb91edf2650478ebe86f35c1b41b60c07b50f0bebaee09ff0edd1d41b9a03eb80ac91e366dd86abf65072d10f86bd7f"
        },
        {
            "transaction_type": "rooch",
            "sequence_number": 1,
            "sender": "0x583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e",
            "action_type": "functioncall",
            "action": {
                "function_call": {
                    "function_id": "583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e::fixed_supply_coin::faucet",
                    "ty_args": [],
                    "args": []
                }
            },
            "raw": "0x583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e0100000000000000d7af34010000000000e1f5050000000001583b6ddb6b28d7b69ae968395e542d675c855d6d6d4644c5dc1426bbafd4b01e1166697865645f737570706c795f636f696e06666175636574000000000000000000006100a25a8b368234cdd8ed03ba6c595c067ab65812f3b2dc38741d2afb8ebb1c53167126977fdab5fb19ea5eb2314cbca20d75ea13be513cb54419e7f3811ece400407b50f0bebaee09ff0edd1d41b9a03eb80ac91e366dd86abf65072d10f86bd7f"
        }
    ],
    "id": 101
}
```

### CODE REVIEW

**Comment by baichuan3** [comment]
@wow-sven  The flow of obtaining transaction list through Explorer or Dashboard is above.

**Comment by baichuan3** [comment]
> Rooch's tx order is determined before execution. How about keeping both `getTransactionByTxOrder` and `getTransaction`(ByHash)? Make the client and SDK simpler. getTransactionByTxOrder also can used to sync tx between rooch nodes.

For now `getTransaction(ByHash)` get one Tx and `getTransactions` get multi Txs, I feel doesn't get your point.

**Comment by baichuan3** [comment]
> Rooch's tx order is determined before execution. How about keeping both `getTransactionByTxOrder` and `getTransaction`(ByHash)? Make the client and SDK simpler. getTransactionByTxOrder also can used to sync tx between rooch nodes.

If Rooch's tx order is determined before execution, we can consider writing sequencer order into TypeTransaction so that we can obtain the transaction list through one API.

**Comment by baichuan3** [comment]
Encounter an CI error due to `dashboard.rs` , help to have a look @wow-sven

**Comment by jolestar** [comment]
> For now `getTransaction(ByHash)` get one Tx and `getTransactions` get multi Txs, I feel doesn't get your point.

We can provide both getTransactionsByTxOrder for scanning the tx list and getTransactions(ByHash) for multi-get by hash.

We already have a `tx_seq_info_mapping` storage that keeps tx_order -> tx_hash mapping, Implementing this on the server side is more efficient.

**Comment by baichuan3** [comment]
> > For now `getTransaction(ByHash)` get one Tx and `getTransactions` get multi Txs, I feel doesn't get your point.
> 
> We can provide both getTransactionsByTxOrder for scanning the tx list and getTransactions(ByHash) for multi-get by hash.
> 
> We already have a `tx_seq_info_mapping` storage that keeps tx_order -> tx_hash mapping, Implementing this on the server side is more efficient.

Already Implementing in `getTransactionsByOrder`, For details, see the Summary at the top.

File: crates/rooch-rpc-api/src/jsonrpc_types/transaction_view.rs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+use crate::jsonrpc_types::{
+    TransactionExecutionInfoView, TransactionSequenceInfoView, TransactionView,
+};
+use moveos_types::transaction::TransactionExecutionInfo;
+use rooch_types::transaction::{TransactionSequenceInfo, TypedTransaction};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone)]
+pub struct TransactionReturn {
```
Comment (jolestar) [inline crates/rooch-rpc-api/src/jsonrpc_types/transaction_view.rs:13]: `TransactionReturn` is not a good name. How about `TransactionResult`? `TransactionWithInfo`?

**Review by jolestar — APPROVED** [review]

File: crates/rooch-rpc-api/src/jsonrpc_types/transaction_view.rs (hunk 1 above)
Comment (jolestar) [reply crates/rooch-rpc-api/src/jsonrpc_types/transaction_view.rs:13]: I merged this PR first. We can discuss the name later.

**Pull request merged**